doc/whats_new/v0.21.rst

```diff
@@ -542,6 +542,12 @@ Multiple modules
 - |Efficiency| Memory copies are avoided when casting arrays to a different
   dtype in multiple estimators. :issue:`11973` by :user:`Roman Yurchak
   <rth>`.
+- |API| Removed all mentions of ``warn_on_dtype=False`` and deprecated
```

Comment:
Please put this under the  

```
:mod:`sklearn.utils`
....................
```

section (create it if it does not exist).

Also you can remove the first part *Removed all mentions of ``warn_on_dtype=False`` and *

Reply:
done